usecases/recording.cameras.opencv.3/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+version: '2'
+# Please note that docker-compose does not prescribe a startup order
+
+#This docker-compose file makes camera-proxy and odrecorderh264 run in headless mode, i.e., recording videos without display. In this mode, the debug parameter of proxy-camera in the configuration file needs to be set to 0. If a displayer is available, it is possible to view what is being recorded by (1) setting the debug parameter of proxy-camera in the configuration file to 1; and (2) add the following to services proxy1, proxy2, and proxy3:
+
+#environment:
+#- DISPLAY=$DISPLAY
+#volumes:
+#- /tmp/.X11-unix:/tmp/.X11-unix
+
+############################
+#Run the following to replay the recordings:
+#docker run -ti --rm --net=host -v $HOME/recordings:/opt/recordings seresearch/opendavinci-ubuntu-16.04-complete:latest /opt/od4/bin/odsupercomponent --cid=111 --verbose=1  --configuration=/opt/recordings/configuration
+
+#docker run -ti --rm --net=host --ipc=host --user=root -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix -v $HOME/recordings:/opt/recordings seresearch/opendavinci-ubuntu-16.04-complete:latest /opt/od4/bin/odcockpit --cid=111
+
+#docker run -ti --rm --net=host --ipc=host --user=root -v $HOME/recordings:/opt/recordings -w /opt/recordings seresearch/opendavinci-ubuntu-16.04-complete:latest /opt/od4/bin/odplayerh264 --cid=111
+
+services:
+    #odsupercomponent
+    odsupercomponent:
+        build: .
+        image: seresearch/opendavinci-ubuntu-16.04-complete:latest
+        volumes:
+        - ~/recordings:/opt/recordings
+        command: "/opt/od4/bin/odsupercomponent --cid=${CID} --verbose=1  --configuration=/opt/recordings/configuration"
+        
+    #camera 1
+    proxy1:
+        build: .
+        image: seresearch/opendlv-core-on-opendavinci-ubuntu-16.04-complete:latest
+        ipc: host
+        user: root
```

Comment:
@HangYin2015 All user: root entries needs to be removed with group_add: video, right?

https://github.com/docker/compose/pull/3709